dev/com.ibm.ws.security.social/resources/OSGI-INF/l10n/metatype.properties

```diff
@@ -238,3 +238,23 @@ discoveryPollingRate.desc=Rate in milliseconds at which the OpenID Connect clien
 
 forwardLoginParameter=Forward login parameter
 forwardLoginParameter.desc=Specifies a comma-separated list of parameter names to forward to the OpenID Connect provider. If a protected resource request includes one or more of the specified parameters, the OpenID Connect client will include those parameters and their values in the authorization endpoint request to the OpenID Connect provider.
+
+userApiType=User API specification
+userApiType.desc=Indicates the specification to use for the user API.
+
+userApiType.basic=Indicates that the user API can be directly invoked with an HTTP GET request.
+# Do not translate "Kubernetes", "TokenReview"
+userApiType.kube=Indicates that the user API is a Kubernetes TokenReview API
```

Comment:
Add a period at the end of "Indicates that the user API can be directly started with an HTTP GET request".